### ISSUE TITLE
The service "doctrine.orm.listeners.lock_store_schema_listener" has a dependency on a non-existent service "semaphore". 

### ISSUE THREAD

**Issue by alexander-schranz** [symfony/symfony#58383]
### Symfony version(s) affected

7.2.0-dev (not happening on stable versions v7.1.5)

### Description

Since some week we got on the `@dev` version CI the following error:

```
Could not execute command "/usr/bin/php8.3 /home/runner/work/sulu/sulu/bin/../src/Sulu/Bundle/TestBundle/Resources/app/console doctrine:schema:update --force", got exit code "1": 
In CheckExceptionOnInvalidReferenceBehaviorPass.php line 116:
                                                                               
  The service "doctrine.orm.listeners.lock_store_schema_listener" has a depen  
  dency on a non-existent service "semaphore".                                 
```

The error does not appear on `@stabe` dependencies. The whole output of the error can currently be found here: https://github.com/sulu/sulu/actions/runs/11021611932/job/30609077475?pr=759

As seen in the output version step `sysvsem` php extension is installed. I can not really find out where `semaphore` service is defined and is coming from.

### How to reproduce

```
git clone git@github.com:sulu/sulu.git sulu
cd sulu
composer config minimu-stability dev
composer update

composer bootstrap-test-env
```

### Additional Context

<details>
<summary>Stack Trace</summary>

```
In CheckExceptionOnInvalidReferenceBehaviorPass.php line 116:

  [Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException]
  The service "doctrine.orm.listeners.lock_store_schema_listener" has a depen
  dency on a non-existent service "semaphore".


Exception trace:
  at /private/tmp/sulu/vendor/symfony/dependency-injection/Compiler/CheckExceptionOnInvalidReferenceBehaviorPass.php:116
 Symfony\Component\DependencyInjection\Compiler\AbstractRecursivePass@anonymous\/private/tmp/sulu/vendor/symfony/dependency-injection/Compiler/CheckExceptionOnInvalidReferenceBehaviorPass.php:101$8b->processValue() at /private/tmp/sulu/vendor/symfony/dependency-injection/Compiler/CheckExceptionOnInvalidReferenceBehaviorPass.php:123
 Symfony\Component\DependencyInjection\Compiler\CheckExceptionOnInvalidReferenceBehaviorPass->throwServiceNotFoundException() at /private/tmp/sulu/vendor/symfony/dependency-injection/Compiler/CheckExceptionOnInvalidReferenceBehaviorPass.php:83
 Symfony\Component\DependencyInjection\Compiler\CheckExceptionOnInvalidReferenceBehaviorPass->processValue() at /private/tmp/sulu/vendor/symfony/dependency-injection/Compiler/AbstractRecursivePass.php:84
 Symfony\Component\DependencyInjection\Compiler\AbstractRecursivePass->processValue() at /private/tmp/sulu/vendor/symfony/dependency-injection/Compiler/CheckExceptionOnInvalidReferenceBehaviorPass.php:48
 Symfony\Component\DependencyInjection\Compiler\CheckExceptionOnInvalidReferenceBehaviorPass->processValue() at /private/tmp/sulu/vendor/symfony/dependency-injection/Compiler/AbstractRecursivePass.php:93
 Symfony\Component\DependencyInjection\Compiler\AbstractRecursivePass->processValue() at /private/tmp/sulu/vendor/symfony/dependency-injection/Compiler/CheckExceptionOnInvalidReferenceBehaviorPass.php:48
 Symfony\Component\DependencyInjection\Compiler\CheckExceptionOnInvalidReferenceBehaviorPass->processValue() at /private/tmp/sulu/vendor/symfony/dependency-injection/Compiler/AbstractRecursivePass.php:84
 Symfony\Component\DependencyInjection\Compiler\AbstractRecursivePass->processValue() at /private/tmp/sulu/vendor/symfony/dependency-injection/Compiler/CheckExceptionOnInvalidReferenceBehaviorPass.php:48
 Symfony\Component\DependencyInjection\Compiler\CheckExceptionOnInvalidReferenceBehaviorPass->processValue() at /private/tmp/sulu/vendor/symfony/dependency-injection/Compiler/AbstractRecursivePass.php:45
 Symfony\Component\DependencyInjection\Compiler\AbstractRecursivePass->process() at /private/tmp/sulu/vendor/symfony/dependency-injection/Compiler/CheckExceptionOnInvalidReferenceBehaviorPass.php:39
 Symfony\Component\DependencyInjection\Compiler\CheckExceptionOnInvalidReferenceBehaviorPass->process() at /private/tmp/sulu/vendor/symfony/dependency-injection/Compiler/Compiler.php:73
 Symfony\Component\DependencyInjection\Compiler\Compiler->compile() at /private/tmp/sulu/vendor/symfony/dependency-injection/ContainerBuilder.php:814
 Symfony\Component\DependencyInjection\ContainerBuilder->compile() at /private/tmp/sulu/vendor/symfony/http-kernel/Kernel.php:498
 Symfony\Component\HttpKernel\Kernel->initializeContainer() at /private/tmp/sulu/vendor/symfony/http-kernel/Kernel.php:743
 Symfony\Component\HttpKernel\Kernel->preBoot() at /private/tmp/sulu/vendor/symfony/http-kernel/Kernel.php:120
 Symfony\Component\HttpKernel\Kernel->boot() at /private/tmp/sulu/vendor/symfony/framework-bundle/Console/Application.php:177
 Symfony\Bundle\FrameworkBundle\Console\Application->registerCommands() at /private/tmp/sulu/vendor/symfony/framework-bundle/Console/Application.php:69
 Symfony\Bundle\FrameworkBundle\Console\Application->doRun() at /private/tmp/sulu/vendor/symfony/console/Application.php:167
 Symfony\Component\Console\Application->run() at /private/tmp/sulu/src/Sulu/Bundle/TestBundle/Resources/app/console:27
```

</details>

**Comment by xabbuh** [comment]
Can you check if it works without c8137fe5623b1ed02b8d56737db44e88c222b95c?

**Comment by xabbuh** [comment]
And if that's the case does #58445 fix it?

**Comment by HypeMC** [comment]
Didn't see this one earlier, this was definitely caused by #58249 and fixed with #58445.

**Comment by alexander-schranz** [comment]
Looks fine again in our CI. thx all!
